### DISCUSSION TITLE
Orchil Questions

### DISCUSSION THREAD

**Discussion by anony-0** [ChatTheatre/Community#9] [General]
I have about a zillion questions to ask, but figured I'd start with some basic stuff.

First questions are about orchil ... mainly what are its limitations/issues compared to zealotry? I've been out of the loop for a while, so didn't really play much with it when it was first released.

Anyway, from what I can guess, it has some theme issues and doesn't do the macro thing? Or has that been fixed by now?

How are most games handling theme issues, just using a default theme for their specific games? In my case I always preferred bland black/white default-ness, so at least so far orchil seems fine with me theme-wise. But I expect not everyone has such bland theme tastes as I do.

**Comment by noahgibbs** [comment]
You are 100% correct on "has some theme issues and doesn't do the macro thing."

**Comment by anony-0** [comment]
Last night I logged into CM for the first time in ages and... well, yeah, orchil has some issues. It was a choice of retina destroying themes or going back to zealotry, but then the sidebar art is missing.

In AOE Orchil seems to play nicer... is this because they lock into a specific black text on white background theme?

And in Lazarus I have been playing with things a little and so far Orchil has been playing relatively nice... but again, locked into a dull theme.  But not being live and having no players tends to cut down on the complaints. I have been having big map/popup issues, but not sure if that is due to Orchil or some redirects that need fixing.

Then there are macros... ugh... really, no macros? Any workarounds at all?

**Reply by noahgibbs** [reply]
Oh hey, this is worth an update - the latest Orchil on GitHub now has both support for Zealotry-type server-side themes and for macros. I don't know when/if various games will pick up these changes, but they exist on GitHub.
